BLSQ/iaso
# fix: autocomplete render option api changes

Fix autocomplete render option api changes in MUI5

Related JIRA tickets : IA-2656

## Self proofreading checklist


## Changes

#### Fix autocompletes

From my understanding MUI Autocomplete signature changed in mui 5
from `(option) => ....`
to `(props, option) => ....`

this fixes the DHIS2 and Form version autocomplete

#### Fix seed command

Apparently the upgrades of the latest dependencies, broken the seed command.
Xls file can't be parsed by pyxform, so I saved them as xlsx and adapted the code.
Note that the setuper doesn't cover the dhis2 integration yet.

## How to test

seed a dhis2 env
```
docker-compose run --rm iaso manage seed_test_data --mode=seed --dhis2version=2.38
```
then login into iaso
go to the dhis2 mapping screen edit an existing one for Quality PCA form 2.38.5.1
select a question in the tree, try to search for 'treated'

Also go to the mapping list create a fresh mapping
Press create
the auto complete for form version and datasets should work




## Print screen / video


![image](https://github.com/BLSQ/iaso/assets/371692/8a1750f1-cb96-43f9-9e42-ee98314aa958)

![image](https://github.com/BLSQ/iaso/assets/371692/bf459a43-f7d6-481b-90c8-4e71af01592d)


## Notes
